rts/Rendering/GL/myGL.cpp

```diff
@@ -135,7 +135,7 @@ static bool GetVideoMemInfoATI(GLint* memInfo)
 	// these are not disjoint, don't sum
 	for (uint32_t param: {/*GL_VBO_FREE_MEMORY_ATI,*/ GL_TEXTURE_FREE_MEMORY_ATI/*, GL_RENDERBUFFER_FREE_MEMORY_ATI*/}) {
 		glGetIntegerv(param, &memInfo[0]);
-
+		if (memInfo[2] < 0) memInfo[2] = ( 1 << 27 ) + memInfo[2]; // workaround
```

Comment:
Can you add a comment this is a workaround for amd gpus that might return negative sizes?

Reply:
Sure, is it looks okay now?

Reply:
Yes, thanks!